TLM/TLM/TLM.csproj

```diff
@@ -74,7 +74,7 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Benchmark|AnyCPU'">
     <OutputPath>bin\Benchmark\</OutputPath>
-    <DefineConstants>QUEUEDSTATS;SPEEDLIMITS;ROUTING;JUNCTIONRESTRICTIONS;VEHICLERESTRICTIONS;ADVANCEDAI;CUSTOMTRAFFICLIGHTS;BENCHMARK</DefineConstants>
+    <DefineConstants>QUEUEDSTATS;SPEEDLIMITS;ROUTING;JUNCTIONRESTRICTIONS;VEHICLERESTRICTIONS;ADVANCEDAI;CUSTOMTRAFFICLIGHTS;BENCHMARK;ENABLE_PROFILER</DefineConstants>
```

Comment:
What does `ENABLE_PROFILER` do? The code in the other file checks for `BENCHMARK`?

Reply:
By itself it does nothing, but its necessary to allow you to actually see numbers in unity profiler if you create Profiler.BeginSample("..") .. Profiler.EndSamle() sections.

Regarding the other file, this is old stuff that we removed some time ago, but due to the #if it did not show up as a compiler error xD and was forgotten.